CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/DatabaseRequestHandler.swift

```diff
@@ -206,7 +206,8 @@ public class DatabaseRequestHandler {
             try database.inBatch {
                 for doc in documents {
                     let id = doc.key
-                    let data: Dictionary<String, Any> = doc.value
+                    var data: Dictionary<String, Any> = doc.value
+                    data.removeValue(forKey: "_id")
```

Comment:
wondering in what cases you get _id from pytest. As u have update on test, we should not be getting _id from the test. It is good to update on the test side to avoid sending _id rather than doing on app to be safe side and have the tests to run on right way.

Reply:
Around 100+ cbl test cases we are using _id , to avoid updating all the tests as we decided to make app changes, I made it as discussed and confirmed on Friday and Thurday.

Reply:
are you sure here is the only location removing _id key is required? in this class, at least updateDocuments and saveDocuments involve doc keys (may or may not include _id)

Reply:
Updated as per the Sri's suggestion. Other api's do not require changes , since we made changes to create_doc and create_bulk_doc methods in the test framework